app/home/heading.tsx

```
@@ -0,0 +1,51 @@
+      animate={{ opacity: 1, y: 0, scale: 1 }}
+      transition={{ duration: 0.8, ease: "easeOut" }}
+    >
+      <p className="text-xl font-normal leading-normal text-copy mb-4">
+        Hey, I'm Andel 👋
+      </p>
+      <h1 className="mb-6 text-balance max-w-3xl text-3xl font-semibold leading-tight md:text-5xl text-copy md:leading-[1.1]">
+        I'm an independent Web Developer based in Trinidad and Tobago.
+      </h1>
+      <p className="mb-8 text-text-muted text-balance max-w-2xl text-xl md:text-2xl font-normal leading-normal ">
+        As a builder at heart, I'm sharing products with the world like:{" "}
+        <a target="_blank" href="https://taack.app" className="text-primary">
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)